apps/b2b/components/Auth/RegistrationForm.tsx

```diff
@@ -0,0 +1,124 @@
+import { gql, useMutation } from '@apollo/client'
+import { yupResolver } from '@hookform/resolvers/yup'
+import ErrorMessage from 'components/ErrorMessage'
+import Input from 'components/Input'
+import Link from 'next/link'
+import { useForm } from 'react-hook-form'
+import { Button, Typography } from 'ui'
+import * as yup from 'yup'
+
+const FormSchema = yup.object({
+  name: yup.string().required().label('Name'),
```

Comment:
do we really just want one name? or should we go for first name and last name instead?

Reply:
I'll add up an issue for this 👍